cmake/Modules/MacroQbtCommonConfig.cmake

```diff
@@ -38,7 +38,7 @@ macro(qbt_common_config)
         cxx_variable_templates
     )
 
-    set(QBT_FULL_VERSION "${qBittorrent_VERSION}${QBT_VER_STATUS}")
+    set(QBT_FULL_VERSION "${qBittorrent_VERSION_MAJOR}.${qBittorrent_VERSION_MINOR}.${qBittorrent_VERSION_PATCH}${QBT_VER_STATUS}")
```

Comment:
QMake builds version string dynamically according to "build" value:
```qmake
PROJECT_VERSION = $${VER_MAJOR}.$${VER_MINOR}.$${VER_BUGFIX}

!equals(VER_BUILD, 0) {
    PROJECT_VERSION = $${PROJECT_VERSION}.$${VER_BUILD}
}

PROJECT_VERSION = $${PROJECT_VERSION}$${VER_STATUS}
```
Why not follow it here?